crates/holochain_p2p/src/test.rs

```diff
@@ -330,53 +330,6 @@ mod tests {
         r_task.await.unwrap();
     }
 
-    #[tokio::test(flavor = "multi_thread")]
```

Comment:
This test was accidentally merged with the previous PR 😬

Reply:
WE DEFINITELY ALL NOTICED AND THOUGHT IT WAS DELIBERATE AND A REALLY GREAT TEST